linux_os/guide/system/accounts/accounts-restrictions/password_storage/accounts_password_last_change_is_in_past/tests/all_password_change_time_in_past.pass.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# packages = passwd
```

Comment:
```suggestion
```
Now we can remove this header since the `passwd` command is no longer used.

Reply:
@teacup-on-rockingchair , could you remove here too, please? This should save some seconds during the test scenarios when this package is not installed.

Reply:
I am a bad clicker obviously 🤦